packages/examples/pages/users.tsx

```
@@ -6,13 +6,37 @@ import { useQuery } from 'react-query';
 
 import { UserTable } from '../components/UserTable';
 
+const useOrgQuery = () => {
+  const nile = useNile();
+  const { isLoading: userLoading, data: user } = useQuery('me', () =>
+    nile.getMe()
+  );
+
+  const { isLoading: orgLoading, data: orgs } = useQuery('organizations', () =>
+    nile.listOrganizations({})
+  );
+  const currentUserEmail = user?.email;
+
+  const orgId = React.useMemo(() => {
+    const org = orgs?.filter((org) => org.name === currentUserEmail);
+    if (org && org.length === 1) {
```

if not this, then it fails silently :(